crossbeam-utils/Cargo.toml

```diff
@@ -25,6 +25,9 @@ alloc = []
 cfg-if = "0.1.2"
 lazy_static = { version = "1.1.0", optional = true }
 
+[target.'cfg(loom)'.dependencies]
```

Comment:
I'm surprised that this works. (I felt it was a bug, but actually, [only some things like `cfg(feature)` don't seem to work.](https://github.com/rust-lang/cargo/pull/7660/files#diff-2ab1ff723e1d553d9d5382510b10cfe6R75-R97))

Reply:
Yeah, I was surprised too! I think the thinking here is that you can only use `cfg()` in this spot for `cfg` that are available from the very early stages of compilation (like `RUSTFLAGS`). Or something. Things that are only available later (like after figuring out what features the crate will be compiled by) can't be used.